spyder/plugins/console/widgets/internalshell.py

```diff
@@ -129,34 +129,30 @@ def end_input(self, cmd):
 class InternalShell(PythonShellWidget):
     """Shell base widget: link between PythonShellWidget and Interpreter"""
 
-    status = Signal(str)
-    refresh = Signal()
-    go_to_error = Signal(str)
-    focus_changed = Signal()
+    sig_refreshed = Signal()
+    sig_show_status_requested = Signal(str)
+    sig_go_to_error_requested = Signal(str)
+    sig_focus_changed = Signal()

```

Comment:
Please add a comment above these signals to describe what they do.

Reply:
Fixed